plugins/Provider/API.php

```diff
@@ -29,7 +29,6 @@ public function getProvider($idSite, $period, $date, $segment = false)
         Piwik::checkUserHasViewAccess($idSite);
         $archive = Archive::build($idSite, $period, $date, $segment);
         $dataTable = $archive->getDataTable(Archiver::PROVIDER_RECORD_NAME);
-        $dataTable->filter('Sort', array(Metrics::INDEX_NB_VISITS));
```

Comment:
No longer needed since we do now always sort in GenericFilter. So we do no longer sort the entire table twice